frontend/packages/db-structure/src/schema/overrideDbStructure.ts

```diff
@@ -43,9 +50,11 @@ export type DBOverride = v.InferOutput<typeof dbOverrideSchema>
  * Applies override definitions to the existing DB structure.
  * This function will:
  * 1. Add new tables from addTables
- * 2. Apply overrides to existing tables (e.g., replacing comments)
- * 3. Add new columns to existing tables
- * 4. Add new relationships
+ * 2. Apply overrides to existing tables
+ * 2.1. Replace comments
+ * 2.2. Add new columns
+ * 2.3. Add new constraints
+ * 3. Add new relationships
```

Comment:
I arranged it a little bit.
I think `adding new columns` step is a part of `applying overrides to existing tables`. Does that align with your intention?